ocs_ci/utility/aws.py

```diff
@@ -262,50 +269,75 @@ def detach_and_delete_volume(self, volume, timeout=120):
             delete_response
         )
 
-    def stop_ec2_instance(self, instance_id, wait=False):
+    def stop_ec2_instances(self, instances, wait=False, force=True):
         """
         Stopping an instance
 
         Args:
-            instance_id (str): ID of the instance to stop
+            instances (dict): A dictionary of instance IDs and names to stop
             wait (bool): True in case wait for status is needed,
                 False otherwise
+            force (bool): True for force instance stop, False otherwise
 
-        Returns:
-            bool: True in case operation succeeded, False otherwise
         """
-        res = self.ec2_client.stop_instances(
-            InstanceIds=[instance_id], Force=True
-        )
+        instance_ids = [key for key in instances.keys()]
+        instance_names = [val for val in instances.values()]
+        logger.info(f"Stopping instances {instance_names} with Force={force}")
+        assert self.ec2_client.stop_instances(
```

Comment:
stop_instances() always returns a list containing information about the instance(s) status(es), and thus will always evaluate to True.
I think it'll be best to parse the returning dictionary, and print+analyze the results accordingly.
More here:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.stop_instances